Tests/Testo/Invoice/InvItem/InvItemServiceTest.php

```
@@ -0,0 +1,1175 @@
+        /** @var \Mockery\Expectation $e */
+        $e = $reader->shouldReceive('getIterator');
+        $e->andReturn((static function () use ($items) {
+            yield from $items;
+        })());
+        return $reader;
+    }
+
+    private function makeService(
+        ?InvItemAllowanceChargeRepository $aciiR = null,
+        ?InvItemRepository $repository = null,
+        ?InvRepository $iR = null,
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $iR. Configured minimum length is 3. (undefined)

(ShortVariable)